### PR TITLE
[NOTIFICATION] Create docker and folder structure for notification service

### PR DESCRIPTION
**Pull request by aacgn**
Solves #20 and #22

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 pgdata-core
 pgdata-preferences
 pgdata-order
+pgdata-notification
```
Comment (aacgn) [inline .gitignore]: Change that to pgdata-*

File: notification-service/src/app.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import 'reflect-metadata';
+import DatabaseConfig from './config/database';
+import * as Express from 'express';
+import * as bodyParser from 'body-parser';
+import NotificationRouter from './routes/notification.route'
+
+class Notification {
```
Comment (aacgn) [inline notification-service/src/app.ts]: Change this name for something like NotificationApp, just to follow the pattern of previous services

File: notification-service/src/routes/notification.route.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import { NotificationController } from "../controllers/notification.controller";
+
+
+function NotificationRouter(): Router {
+    const router = Router();
+
+    router.post("/create", NotificationController.createNotification);
+    router.get("/", NotificationController.getAllNotifications);
+    router.get("/:id", NotificationController.getNotificationById);
+    router.get("/user/:userId", NotificationController.getNotificationByUserId);
```
Comment (aacgn) [inline notification-service/src/routes/notification.route.ts]: Follow the previous pattern, and change this one for "users"

File: notification-service/src/app.ts (hunk 2 above)
Comment (eduardosm7) [reply notification-service/src/app.ts]: Done

File: .gitignore (hunk 1 above)
Comment (eduardosm7) [reply .gitignore]: Done

**Review by viniciusgiles — APPROVED** [review]

**Pull request merged**